packages/l/libmpdclient/xmake.lua

```diff
@@ -8,6 +8,8 @@ package("libmpdclient")
     on_install("linux", function (package)
         import("package.tools.meson").install(package)
```

Comment:
We need to use `package:config("shared")` to configure to distinguish static/shared library.

Reply:
I'm not exactly what sure what I'm supposed to do here to be honest.
I've tried various combinations of adding `package:config("shared")` and I don't think I'm getting it.

Reply:
like these:

https://github.com/xmake-io/xmake-repo/blob/a8fa2de0c922e364a74921b8e118fce229a4e04e/packages/l/libisl/xmake.lua#L14-L15

https://github.com/xmake-io/xmake-repo/blob/a8fa2de0c922e364a74921b8e118fce229a4e04e/packages/l/libogg/xmake.lua#L19

But I don't know what configuration parameters should be passed to meson to set static/shared, and I don't know meson very well.

```
        local configs = {} -- add static/shared configs?
        import("package.tools.meson").install(package, configs)
```

Reply:
https://github.com/mesonbuild/meson/issues/2910 maybe -Ddefault_library=static and -Ddefault_library=shared

Reply:
I think I got it now.